app/main.py

```diff
@@ -1,4 +1,55 @@
-KNIGHTS = {
+from __future__ import annotations
+
+from app.knights.knight import Knight
+
+
+# creation of knights
+
+
+def knights_config(knights: dict[str, any]) -> dict:
+    knights_dict = {}
+    for knight_name, knight_data in knights.items():
+        knight = Knight(
+            knight_data["name"],
+            knight_data["power"],
+            knight_data["hp"],
+        )
+        knight.protection = 0
+        knight.apply_armour(knight_data["armour"])
+        knight.apply_weapon(knight_data["weapon"])
+        knight.apply_potion(knight_data["potion"])
```

Comment:
You can call this methods directly in init
And use dict comprehension instead of for loop